tests/test_embed/test_interpreter.cpp

```diff
@@ -278,7 +278,7 @@ TEST_CASE("Reload module from file") {
     test_module.close();
 
     // Reload the module
-    module.reload();
-    result = module.attr("test")().cast<int>();
+    module_.reload();
+    result = module_.attr("test")().cast<int>();
```

Comment:
This is an example of a test variable renamed, just in case. It is the first item on a logical line, so by definition it might break on some compilers in C++20 mode.

Reply:
Interesting that we didn't get any VC++ failures, yet, then? Or are we not yet testing C++20?

Reply:
We are not testing C++20 anywhere, it's always 11+17, or 14.

Reply:
FWIW, using g++ (Debian 9.3.0-14) 9.3.0 or clang version 9.0.1-12 with `-std=c++2a` I don't get any errors.
Irrespective, I like this variable change 100%.